spring-cloud-data-admin/src/main/java/org/springframework/cloud/data/admin/config/AdminConfiguration.java

```diff
@@ -86,7 +86,12 @@ public ModuleRegistry moduleRegistry() {
 	protected static class LocalConfig {
 
 		@Bean
-		public ModuleDeployer moduleDeployer(ModuleLauncher moduleLauncher) {
+		public ModuleDeployer longRunningModuleDeployer(ModuleLauncher moduleLauncher) {
```

Comment:
okay - that's more convincing... can we at least decouple from the diego terms by using something simpler such as "process" and "task" then?